docs/whats-new/dotnet-aspire-9.2.md

```diff
@@ -0,0 +1,430 @@
+
+## App Host
+
+### 🚧 project file changes
+
+The .NET Aspire app host project file no longer requires the `IsAspireHost` property. This property was moved to the `Aspire.AppHost.Sdk` SDK, therefore, you can remove it from your project file. For more information, see [dotnet/aspire issue #8144](https://github.com/dotnet/aspire/pull/8144).
+
+### Custom Resource URLs
+
+Resources can now define custom URLs! This makes it easier to build custom experiences for your resources. For example, you can define a custom URL for a database resource that points to the database management console. This makes it easier to access the management console directly from the dashboard, you can even give it a friendly name.
+
+```C#
+var catalogDb = builder.AddPostgres("postgres")
+                    .WithDataVolume()
+                    .WithPgAdmin(resource =>
+                    {
+                        resource.WithUrlForEndpoint("http", u => u.DisplayText = "PG Admin");
+                    })
+                    .AddDatabase("catalogdb");
+```
+
+The above code sets the display text for the `PG Admin` URL to `PG Admin`. This makes it easier to access the management console directly from the dashboard.
+
+### WithHttpCommand
+
+Adds WithHttpCommand(), which lets you define a resource command that sends an HTTP request to your app during development. Useful for triggering endpoints like seed or reset from the dashboard.
+
+```c#
+if (builder.Environment.IsDevelopment())
+{
+    var resetDbKey = Guid.NewGuid().ToString();
+    catalogDbApp.WithEnvironment("DatabaseResetKey", resetDbKey)
+                .WithHttpCommand("/reset-db", "Reset Database",
+                    commandOptions: new()
+                    {
+                        Description = "Reset the catalog database to its initial state. This will delete and recreate the database.",
+                        ConfirmationMessage = "Are you sure you want to reset the catalog database?",
+                        IconName = "DatabaseLightning",
+                        PrepareRequest = requestContext =>
+                        {
+                            requestContext.Request.Headers.Add("Authorization", $"Key {resetDbKey}");
+                            return Task.CompletedTask;
+                        }
+                    });
+}
+```
+
+### ConnectionStringResource
+
+We've introduced a new ConnectionStringResource type that makes it easier to build dynamic connection strings without defining a separate resource
+type. This makes it easier to work with build dynamic parameterized connection strings.
+
+   ```c#
+   var apiKey = builder.AddParameter("apiKey");
+   var cs = builder.ApplicationBuilder.AddConnectionString("openai", csb =>
```

Comment:
Should probably get rid of `ApplicationBuilder` here. Assume this is just copy pasta.